content/guides/integrations/add-wandb-to-any-library.md

```diff
@@ -31,7 +31,7 @@ Below we cover best tips and best practices when the codebase you are working on
 
 Before you get started, decide whether or not to require W&B in your library’s dependencies:
 
-#### Require W&B On Installation
+#### Require W&B on installation
```

Comment:
This and the next heading seem odd to me. It seems like you need to choose whether to require it, and depending on your choice, you need to take an action either way. What if these were collapsed into a single section "Add W&B as a dependency" and start with "If you add it to your dependencies file, it is required. To make it optional, <foo>.

Second, in the next section, I'd change the A and B to list items, or sub headings (but we are getting deep). You'll need to indent your code blocks two stops, I believe.